src/routes/blog/index.svelte

```diff
@@ -15,22 +15,31 @@
 </script>
 
 <svelte:head>
-  <title>acmCSUF / README</title>
+  <title>Blog / ACM at CSUF</title>
 </svelte:head>
 
 <Spacing --min="175px" --med="200px" --max="200px" />
 
-<section>
+<section class="main-header">
   <img src="assets/readme-logomark.svg" alt="README by acmCSUF" />
```

Comment:
I have a feeling that referencing the SVG relatively on the `/blog` page might give us varying results.
```suggestion
  <img src="/assets/readme-logomark.svg" alt="README by acmCSUF" />
```

Reply:
resolving this cus it seems to work fine

Reply:
Thank you